src/theme/DocItem/Layout/styles.module.css

```diff
@@ -0,0 +1,10 @@
+.docItemContainer header + *,
+.docItemContainer article > *:first-child {
+  margin-top: 0;
+}
+
+@media (min-width: 997px) {
```

Comment:
out of curiosity, why 997px?

Reply:
No clue tbh. I just swizzled to add the banner, the rest is just how Docusaurus implemented the component.